styles.css

```diff
@@ -91,3 +86,211 @@ ul {
   align-items: center;
   margin: auto;
 }
+
+/* css for the second part of the project */
+
+.works {
+  display: grid;
+  text-align: center;
+  justify-content: center;
+}
+
+.recent-works {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.img-wrapper {
+  height: 252px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+}
+
+.card {
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  flex-direction: column;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  height: 474px;
+  width: 327px;
+}
+
+ul {
+  padding-left: 0;
+}
+
+.card-name {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  color: #3a4a42;
+  align-items: center;
+  justify-content: center;
+}
+
+.technologies {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  list-style-type: none;
+  gap: 15px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  color: #3a4a42;
+}
+
+.tech-tag {
+  background: #ebf0ee;
+  border-radius: 4px;
+  padding: 8px 12px;
+}
+
+.see-project {
+  padding: 12px;
+  gap: 10px;
+  width: 122px;
+  height: 48px;
+  background: #36b37f;
+  border-radius: 4px;
+  color: #fff;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  outline: 0;
+  border: 0;
+}
+
+.see-project:hover {
+  box-shadow: 8px 16px rgba(54, 179, 127, 0.24);
+}
```

Comment:
- [ ] Well done trying to implement the hover effect of buttons, however, the shadow that you applied is larger than it should be, that is because you forgot to add a blur effect, try the code below to fix it 🔽 
```
.see-project:hover {
  box-shadow: 8px 16px 10px rgba(54, 179, 127, 0.24);
}
```